src/components/ViewItem/ViewItem.js

```diff
@@ -132,21 +134,34 @@ class ViewItem extends React.Component {
         );
       },
       'barcode': loan => (<div data-test-item-barcode>{_.get(loan, ['item', 'barcode'])}</div>),
-      'dueDate': loan => (
-        <div data-test-item-due-date>
-          <FormattedDate value={loan.dueDate} />
-          {
-            (loan.dueDateChangedByRecall || loan.dueDateChangedByHold || loan.dueDateChangedByNearExpireUser) && (
-              <Tooltip
-                id="due-date-change-tooltip"
-                text={<FormattedMessage id="ui-checkout.due.date.change" />}
-              >
-                <Icon icon="flag" size="small" />
-              </Tooltip>
-            )
-          }
-        </div>
-      ),
+      'dueDate': loan => {
+        return (
+          <div data-test-item-due-date>
+            <FormattedDate value={loan.dueDate} />
+            {
+              (loan.dueDateChangedByRecall || loan.dueDateChangedByHold || loan.dueDateChangedByNearExpireUser) && (
+                <Tooltip
+                  id="due-date-change-tooltip"
```

Comment:
Probably need to include `loan.id` in Toolip id because this element will be repeated on the page

Reply:
Yes, there is a possibility! Thank you for your insight!!